clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -530,6 +547,7 @@
     "vscode:prepublish": "npm run -S build -- --minify",
     "postinstall": "node -e \"if (process.env.NODE_ENV === 'production'){process.exit(1)} \" && patch-package || echo skip patching",
     "build": "esbuild src/extension.ts --bundle --sourcemap --platform=node --target=node16.17.1 --outdir=dist --external:vscode --format=cjs",
+    "build:web": "esbuild src/web/extension.ts --bundle --sourcemap --platform=node --target=node16.17.1 --outdir=dist --external:vscode --format=cjs",
```

Comment:
Does this actually run in the browser with `---platform=node` and `--format=cjs`?

Reply:
Yes it does. Initially found the build command [here](https://www.roboleary.net/2024/02/16/vscode-ext-esbuild.html), but I have also tested it myself. Outside of the myriad of restrictions that the web version has, there is not too much difference in the core code between it and the installed version as VS Code (installed) uses Electron/Chromium.